### ISSUE TITLE
Cant open second project

### ISSUE THREAD

**Issue by sabadoc** [joeferraro/MavensMate#1016]
* **MavensMate Desktop Version**: 7

* **Editor**: Visual Studio

* **MavensMate Plugin Version**:

* **Platform/Version**:

* **Proxy Details**:

* **Salesforce API Version**:

* **Salesforce Server**:

I just added a second salesforce project to mavensmate desktop, but when i try to launch visual studio from mavens mate, it do nothing. i debugged and inspect it but i didnt found problem.

**Comment by kidtsunami** [comment]
@sabadoc do you have VS Code installed already?

**Comment by sabadoc** [comment]
Yes, i do. I have tried to unistall and install again, but it didn't resolve the problem.

**Comment by kidtsunami** [comment]
And when you run the `Open Project` command in MavensMate for VS Code in VS Code, how many projects does it list?

**Comment by sabadoc** [comment]
it list two projects

**Comment by kidtsunami** [comment]
@sabadoc ok and is the project you're looking for missing from that list?

If it's in the list, can you select it and open it?

If it's not in the list, what happens when you open the folder where the project is located in VS Code?

**Comment by sabadoc** [comment]
No, when i open vs Code, it isn't appear. 
I tried to install sublime text and configure and it runs.
Thanks for the help.
